R/tests/contexts.json

```diff
@@ -0,0 +1,40 @@
+{
+    "contexts": [
+        {
+            "name": "python3-all-local",
```

Comment:
Just have the name as `all-local` and `batch`

Reply:
yep making the change.